src/simplify_reshapes.cpp

```diff
@@ -900,6 +900,38 @@ struct find_scalar_multibroadcast_reshape_or_transpose
     }
 };
 
+// Remove unnecessary preceeding size 1 dims for constants
+struct find_const_multibroadcast
+{
+    auto matcher() const
+    {
+        return match::name("multibroadcast")(
+            match::arg(0)(match::is_constant()(match::used_once()).bind("constant")));
```

Comment:
constant doesnot need to be used_once. `apply()` is not removing or replacing but rather inserting squeeze on it.

Reply:
Thats true but in the unlikely case that somehow its used elsewhere without broadcast, it would duplicate the literal (there would be the original and a copy with the squeezed shape). I dont think this ever happens in reality though, so Im happy to remove that constraint